packages/contentstack-export-to-csv/src/commands/cm/export-to-csv.js

```diff
@@ -59,6 +59,11 @@ class ExportToCsvCommand extends Command {
       multiple: false,
       required: false,
     }),
+    "team-uid": flags.string({
+      description: 'Uid of the team whose user data and stack roles are required',
+      multiple: false,
```

Comment:
@cs-raj, If team-uid is an optional flag, then required and multiple don't need to be included.

Reply:
Removed the fields